docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -35,6 +36,39 @@
 | [v1.0.1](Data_Portal_Release_Notes.md#release-101) | May 18, 2016 |
 
 ---
+## Release 2.0.0
```

Comment:
tbh I've no idea what the expectations are for the release number. internally it'll likely be 2.12.0. would it get confusing down the road if the two release versions are out of sync?